Python/Product/Analysis/LanguageServer/RestTextConverter.cs

```diff
@@ -145,6 +145,7 @@ enum State {
             // they will render as tags and attributes
             line = line.Replace("<", " ").Replace(">", " ").Replace("``", "`"); // Convert double backticks to single.
             // Keep hard line breaks for the preformatted content
+            line = PreserveIndentation(Cleanup(line));
```

Comment:
Can we inject `Replace` calls into `PreserveIndentation` to avoid additional string allocation?